docs/observability/metrics.md

```
@@ -281,6 +281,97 @@ rate(metal_server_reconciliation_total{result=~"error_.*"}[5m])
+
+This is configured via two CLI flags:
+
+| Flag | Source resource | Match key |
+|------|----------------|-----------|
+| `--redfish-metric-labels-from-bmc` | `BMC` resource | resource name == `hostname` label |
+| `--redfish-metric-labels-from-server` | `Server` resource | `spec.bmcRef.name` == `hostname` label |
+
+**Flag format:** `kubernetes-label-key=prometheus-label-name,...`
+
+**Example:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)